spec/features/idv/doc_auth/how_to_verify_spec.rb

```diff
@@ -72,4 +72,135 @@
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        page.refresh
+      end
+
+      it 'should be not be bounced back from hybrid handoff to how to verify' do
+        expect(page).not_to have_current_path(idv_how_to_verify_url)
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+    end
+
+    context 'opt in true at start but false during navigation' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'should be on the how to verify page' do
+        expect(page).to have_current_path(idv_how_to_verify_url)
+      end
+
+      it 'should be on the hybrid handoff page when opt in is false' do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+        page.refresh
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+    end
+  end
+
+  describe 'navigating backwards from How to Verify page in 50/50 state' do
+    context 'Going back from Hybrid Handoff with opt in disabled midstream' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+        complete_how_to_verify_step
+      end
+
+      it 'should be on the how to verify page' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+
+      it 'should be not be bounced back to how to verify with opt in disabled midstream' do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+        page.go_back
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+    end
+
+    context 'Going back from Hybrid Handoff with opt in enabled midstream' do
+      before do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { true }
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { false }
+
+        sign_in_and_2fa_user
+        complete_doc_auth_steps_before_agreement_step
+        complete_agreement_step
+      end
+
+      it 'should be on the hybrid handoff url' do
+        expect(page).to have_current_path(idv_hybrid_handoff_url)
+      end
+
+      it 'should go back to the agreement step from hybrid handoff with opt in toggled midstream' do
+        allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+        page.go_back
+        expect(page).to have_current_path(idv_agreement_url)
```

Comment:
I would expect this to go to the how to verify url.

Reply:
Same. But again this might be a case where going back after toggling the flag breaks backwards navigation. I need to confirm this too.

Reply:
The user's history says they came from agreement, so they go back to agreement. This seems fine to me.